src/SFA.DAS.EmployerIncentives.Commands/Persistence/ApprenticeshipIncentiveDomainRepository.cs

```diff
@@ -88,6 +91,25 @@ public async Task<List<Domain.ApprenticeshipIncentives.ApprenticeshipIncentive>>
                 select _apprenticeshipIncentiveFactory.GetExisting(incentive.Id, incentive)).ToList();
         }
 
+        public async Task<Domain.ApprenticeshipIncentives.ApprenticeshipIncentive> FindByPaymentId(Guid paymentId)
```

Comment:
Is this necessary when this could be done in the use case (i.e. the command handler)?